auacm/app/static/js/controllers/ScoreboardController.js

```diff
@@ -38,26 +37,26 @@ app.controller('ScoreboardController', ['$scope', '$http', '$routeParams',
         var prevSolved = $scope.teams[0].solved;
         var prevTime = $scope.teams[0].solved;
         $scope.teams[0].rank = rank;
-        for (var i = 1; i < $scope.teams.length; i++) {
-            var team = $scope.teams[i];
-            if (team.solved < prevSolved) {
+        for (var j = 1; j < $scope.teams.length; j++) {
```

Comment:
The JavaScript linter in Atom was yelling at me that `i` and `team` had already been declared. I think it was a scoping issue. I can change them to be more apt; I'm not sure if I should change them back though.